### PR TITLE
switch formatter to Prettier

### PR DESCRIPTION
**Pull request by Simon-Laux**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,9 +16,11 @@
     "prebuildify": "prebuildify -t 10.16.0 --napi --strip --postinstall \"node scripts/postinstall.js --prebuild\"",
     "download-prebuilds": "prebuildify-ci download",
     "submodule": "git submodule update --recursive --init",
-    "test": "standard && nyc node test/index.js",
+    "test": "npm run check-formatting && nyc node test/index.js",
     "test-integration": "node test/integration.js",
-    "hallmark": "hallmark --fix"
+    "hallmark": "hallmark --fix",
+    "check-formatting": "prettier --check \"lib/**/*.{ts,tsx}\"",
+    "fix-formatting": "prettier --write \"lib/**/*.{ts,tsx}\""
```
Comment (Jikstra) [inline package.json:23]: I would propose rename "check-formatting" to "lint" and "fix-formatting" to "lint-fix". It's shorter and tab completion (in bash) works better.

**Comment by Jikstra** [comment]
Why does the ci fail only on windows? ^^

**Comment by Simon-Laux** [comment]
The failing on windows could have something to do with line-endings(LF|CRLF), but I haven't looked further into that issue, so this is just a guess.

**Comment by Simon-Laux** [comment]
Looks like I was right with the line-endings

**Pull request merged**